### PR TITLE
Add docs for model extraction

### LINKED ISSUE #467: Document changes to model fitting parameter extraction
Once #458 is merged, the changes need to be added to the appropriate sections of RTD.

### PR DESCRIPTION
**Pull request by javerbukh**
Resolves #467

### CODE REVIEW

**Review by duytnguyendtn — APPROVED** [review]
LGTM!

File: docs/cubeviz/plugins.rst [hunk 1]
```diff
@@ -67,6 +67,20 @@ also saved to a dataset with the label specified in the :guilabel:`Model Label`
...
         Specviz documentation on fitting spectral models.
 
+As mentioned in the `specviz` documentation on model fitting, models can be accessed
+in a jupyter notebook by using the following code::
+
+    cubeviz.get_models(model_label="Model", x=10)
+
+Where the `model_label` parameter identifies which model should be returned and the `x` and
+`y` parameters identify specifically which spaxel coordinates are to be returned. Leaving `x`
+or `y` as `None` will mean that all values are returned. Model parameters can also be accessed
```
Comment (rosteen) [inline docs/cubeviz/plugins.rst:77]:
```suggestion
`y` parameters identify specifically which spaxel fits are to be returned, for models 
applied to every spaxel using the :guilabel:`Apply to Cube` button. Leaving `x` or 
`y` as `None` will mean that the models fit to every spaxel across that axis will be 
returned. Model parameters can also be accessed
```

**Review by rosteen — COMMENTED** [review]
I submitted some suggestions, mostly to add a bit more information for improved (I hope) clarity.

File: docs/cubeviz/plugins.rst (hunk 1 above)
Comment (javerbukh) [reply docs/cubeviz/plugins.rst]: These definitely add clarity, thank you!

**Review by rosteen — APPROVED** [review]

**Pull request merged**